resources/ext.wikibase.facetedsearch.js

```diff
@@ -73,6 +73,16 @@ function onRangeFacetInput( facet, propertyId ) {
 		return;
 	}
 
+	minInput.max = maxInput.value || '';
+	maxInput.min = minInput.value || '';
+	updateErrorState( minInput );
+	updateErrorState( maxInput );
+
+	if ( minInput.validity.valid === false || maxInput.validity.valid === false ) {
```

Comment:
I think it's unnecessary to have explicit checks on `validity.valid`, since it's already defined as a boolean.
![Screenshot_20250117_002128](https://github.com/user-attachments/assets/6358f5d8-ce79-49b4-add1-8fb3f681a84d)

Reply:
I clarified that a few days ago in our Coding Conventions because I spotted a similarly redundant check in PHP

![image](https://github.com/user-attachments/assets/a1c97c77-7255-462d-b244-ae021f200287)